### PR TITLE
doc, vstart.sh: adapt to the generalize AlienStore support

### PR DESCRIPTION
**Pull request by tchaikov**
see also e6ed65db8b4e0a2f8026c2e35a12dd292c5f2b8c

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
https://ceph--42476.org.readthedocs.build/en/42476/dev/crimson/crimson/#crimson

File: doc/dev/crimson/crimson.rst [hunk 1]
```diff
@@ -39,12 +39,30 @@ As you might expect, crimson is not featurewise on par with its predecessor yet.
 object store backend
 --------------------
 
-At the moment ``crimson-osd`` offers two object store backends:
+At the moment, ``crimson-osd`` offers both native and alienized object store
+backends. The native object store backends performs IO using seastar reactor.
```
Comment (rzarzynski) [inline doc/dev/crimson/crimson.rst:43]:
```suggestion
backends. The native object store backends perform IO using seastar reactor.
```

**Review by rzarzynski — APPROVED** [review]
LGTM in general; just 2 nits.

**Comment by tchaikov** [comment]
@rzarzynski fixed and repushed.

**Pull request merged**